Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/167320
+@@||cmp.inmobi.com^|
```

Comment:
@AdamWr Also `||cmp.inmobi.com^` may be removed from Cookies filter or changed to domain specific (maybe element hiding is better).

Reply:
Changed https://github.com/AdguardTeam/AdguardFilters/commit/39eb69e3249055bd78f92db65d255a7076defbc7

Reply:
Okay.
By the way, `inmobi.com` was also blocked by `Base` filter (rule was from `EasyList`), but it seems that it has been moved to `EasyList Cookie` - https://github.com/easylist/easylist/commit/b0a3ff7e7e6efb32c5ee5a5800a800f80e6e8e2c.